src/assets/GlobalStyle.css

```diff
@@ -291,6 +291,14 @@ img {
   background-color: rgb(61, 61, 61) !important;
 }
 
+/* Sticks elements to the top of the page as you scroll down and renders them over other elements */
+.sticky {
+	position: -webkit-sticky;

```

Comment:
This is supposedly there to make it function correctly on Safari, but I do not have a mac to test this on myself